gpt_oss/torch/model.py

```
@@ -443,35 +496,77 @@ def from_checkpoint(
+        else:
+            self.model.eval()   # Disable sampling in eval mode
 
     @torch.inference_mode()
     def generate(self,
                  prompt_tokens: list[int],
                  stop_tokens: list[int],
                  temperature: float = 1.0,
                  max_tokens: int = 0,
-                 return_logprobs: bool = False):
+                 return_logprobs: bool = False,
+                 enable_sampling: bool = None):
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)